disperser/cmd/batcher/main.go

```diff
@@ -102,11 +103,73 @@ func RunBatcher(ctx *cli.Context) error {
 	}, logger, metrics.DispatcherMetrics)
 	asgn := &core.StdAssignmentCoordinator{}
 
-	client, err := geth.NewMultiHomingClient(config.EthClientConfig, gethcommon.HexToAddress(config.FireblocksConfig.WalletAddress), logger)
-	if err != nil {
-		logger.Error("Cannot create chain.Client", "err", err)
-		return err
+	var wallet walletsdk.Wallet
+	var client *geth.MultiHomingClient
+	if !config.KMSKeyConfig.Disable {
+		if config.KMSKeyConfig.KeyID == "" || config.KMSKeyConfig.Region == "" {
+			return errors.New("KMS key ID and region must be specified unless KMS wallet is disabled")
+		}
+		kmsClient, err := kms.NewKMSClient(context.Background(), config.KMSKeyConfig.Region)
```

Comment:
Is it ok to use context.Background()?

Reply:
We can add a timeout there, but I think it's ok during the initialization